csm/cli/schema/cli_setup.json

```diff
@@ -0,0 +1,189 @@
+{
+  "name": "cli_setup",
+  "description": "CLI Setup Command",
+  "sub_commands": [
+    {
+      "name": "init",
+      "description": "Initialize CORTX CLI",
+      "need_confirmation": false,
+      "permissions_tag": "update",
+      "args": [
+        {
+          "flag": "--config",
+          "dest": "config_url",
+          "type": "str",
+          "help": "Config Store URL e.g <type>://<path>."
+        },
+        {
+          "flag": "args",
+          "default": [],
+          "suppress_help": true,
+          "nargs": "?"
+        }
+      ],
+      "comm": {
+        "type": "direct",
+        "target": "csm.cli.conf.init",
+        "method": "execute",
+        "class": "Init",
+        "is_static": false
+      }
+    },
+    {
+      "name": "post_install",
+      "description": "Perform post_install for CORTX CLI",
+      "need_confirmation": false,
+      "permissions_tag": "update",
+      "args": [
+        {
+          "flag": "--config",
+          "dest": "config_url",
+          "type": "str",
+          "required": true,
+          "help": "Config Store URL e.g <type>://<path>."
+        },
+        {
+          "flag": "args",
+          "default": [],
+          "suppress_help": true,
+          "nargs": "?"
+        }
+      ],
+      "comm": {
+        "type": "direct",
+        "target": "csm.cli.conf.post_install",
+        "method": "execute",
+        "class": "PostInstall",
+        "is_static": false
+      }
+    },
+    {
+      "name": "config",
+      "description": "Config CORTX CLI",
+      "need_confirmation": false,
+      "permissions_tag": "update",
+      "args": [
+        {
+          "flag": "--config",
+          "dest": "config_url",
+          "type": "str",
+          "required": true,
+          "help": "Config Store URL e.g <type>://<path>."
+        },
+        {
+          "flag": "--csm-address",
+          "dest": "Address",
+          "type": "str",
+          "default": "127.0.0.1",
+          "help": "CSM URL"
+        },
+        {
+          "flag": "args",
+          "default": [],
+          "suppress_help": true,
+          "nargs": "?"
+        }
+      ],
+      "comm": {
+        "type": "direct",
+        "target": "csm.cli.conf.configure",
+        "method": "execute",
+        "class": "Configure",
+        "is_static": false
+      }
+    },
+    {
+      "name": "reset",
+      "description": "Perform reset for CORTX CLI",
+      "need_confirmation": false,
+      "permissions_tag": "update",
+      "args": [
+        {
+          "flag": "args",
+          "default": [],
+          "suppress_help": true,
+          "nargs": "?"
+        }
+      ],
+      "comm": {
+        "type": "direct",
+        "target": "csm.cli.conf.reset",
+        "method": "execute",
+        "class": "Reset",
+        "is_static": false
+      }
+    },
+    {
+      "name": "refresh_config",
```

Comment:
Refresh Config is not required. This has been removed

Reply:
Removed.